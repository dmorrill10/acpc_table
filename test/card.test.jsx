import React from 'react';
import renderer from 'react-test-renderer';
const Card = require('../dist/card.js');

test('face down card works', () => {
  const patient = <Card/>;
  const tree = renderer.create(patient).toJSON();
  expect(tree).toMatchSnapshot();
});
test('face up card works', () => {
  const patient = <Card rank='A' suit='&spades;'/>;
  const tree = renderer.create(patient).toJSON();
  expect(tree).toMatchSnapshot();
});
