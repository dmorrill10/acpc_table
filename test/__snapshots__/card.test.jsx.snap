exports[`test face down card works 1`] = `
<div
  className="card">
  <div
    className="logo" />
</div>
`;

exports[`test face up card works 1`] = `
<div
  className="card-face-up-s">
  <div
    className="top-left-rank">
    A
  </div>
  <div
    className="suit">
    &spades;
  </div>
  <div
    className="bottom-right-rank">
    A
  </div>
</div>
`;
